ocp_resources/performance_profile.py

```diff
@@ -0,0 +1,112 @@
+from ocp_resources.resource import Resource
+
+
+class PerformanceProfile(Resource):
+    api_group = Resource.ApiGroup.PERFORMANCE_OPENSHIFT_IO
+
+    def __init__(
+        self,
+        additional_kernel_args=None,
+        cpu=None,
+        globally_disable_irq_load_balancing=False,
```

Comment:
Please do not add defaults, we let the API set defaults if any.

Reply:
ok, done